docs/book/src/clusteropenstack/configuration.md

```diff
@@ -107,6 +110,29 @@ Note: Only the [external cloud provider](https://cluster-api-openstack.sigs.k8s.
 
 Note: you need to set `clusterctl.cluster.x-k8s.io/move` label for the secret created from `OPENSTACK_CLOUD_YAML_B64` in order to successfully move objects from bootstrap cluster to target cluster. See [bug 626](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/issues/626) for further information.
 
+## CA certificates
+
+When using an `https` openstack endpoint, providing CA certificates is mandatory.
```

Comment:
nit: You can actually disable it in clouds.yaml with `verify: False` so strictly speaking:

`s/mandatatory/required unless verification is explicitly disabled/`.

Reply:
Thanks! This is now fixed :)